packages/replayio/src/utils/recordings/getRecordingUnusableReason.ts

```diff
@@ -0,0 +1,8 @@
+import { getRecordings } from "./getRecordings";
+
+export function getRecordingUnusableReason(processGroupIdFilter?: string) {
```

Comment:
do we want to keep this parameter optional? it's currently supplied at all times, right?

Reply:
Yes, because the `replayio list` command grabs all recordings. It's only the `replayio record` command that filters by the uid.